src/player/scripts/player.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
```

Comment:
Separate the class definitions into separate files in a src directory. Keep the actual node in scripts. The **src** directory should be for source files that are more or less inactive, **scripts** directory should be for runnable files